src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -32,6 +34,9 @@ public class ParserUtil {
      */
     public static Index parseIndex(String oneBasedIndex) throws ParseException {
         String trimmedIndex = oneBasedIndex.trim();
+        if (trimmedIndex.length() > 4) {
```

Comment:
I realised that checking for the length may not be very ideal.
Previously, if the user enters `profile -d 00001`, the first profile is deleted. however now it shows an error as even though the number is valid it is greater than 5 characters. would it be better to use parseInt or parseLong and check whether it is `< 10000`?

Reply:
I updated the method to correct this. Please help to test again thanks! Also updated the methods and documentation to limit `INDEX` to 1000.